prow/plugins/welcome/welcome.go

```diff
@@ -112,20 +112,29 @@ func handlePR(c client, t plugins.Trigger, pre github.PullRequestEvent, welcomeT
 		return nil
 	}
 
+	org := pre.PullRequest.Base.Repo.Owner.Login
+	repo := pre.PullRequest.Base.Repo.Name
+	user := pre.PullRequest.User.Login
+	pullRequestNumber := pre.PullRequest.Number
+
+	log := c.Logger.WithFields(logrus.Fields{"org": org, "repo": repo, "number": pullRequestNumber})
+
 	// ignore bots, we can't query their PRs
 	if pre.PullRequest.User.Type != github.UserTypeUser {
+		log.Infof("Ignoring bot user '%s', as querying their PRs is not possible", user)
```

Comment:
This should probably be a debug level message.

Reply:
No problem